config/main.py

```diff
@@ -1358,16 +1358,16 @@ def naming_mode_alias():
 #
 # 'syslog' group ('config syslog ...')
 #
-@config.group()
+@config.group('syslog')
 @click.pass_context
-def syslog(ctx):
+def syslog_group(ctx):
```

Comment:
@tsvanduyn IMHO, i think we can try to find a better name for this.  

AFAIK, we already have:
```bash
 sudo show logging
```
How about this:
```bash
sudo config logging server add 1.1.1.1
sudo config logging server del 1.1.1.1
sudo show logging servers
```

What do you think?

Reply:
The method name has changed, but the command syntax did not, due to the change on line 1361, `@config.group('syslog')`. The subcommand is still named "syslog", and the syntax is still:

```
sudo config syslog server add 1.1.1.1
sudo config syslog server del 1.1.1.1
```

Or do you still prefer `logging` over `syslog`?

Reply:
> Or do you still prefer `logging` over `syslog`?
I think we can stay with the original version.